internal/cli/command/cluster/create.go

```diff
@@ -69,6 +70,7 @@ func NewCreateCommand(banzaiCli cli.Cli) *cobra.Command {
 	flags.StringVar(&options.name, "name", "", "Cluster name (overrides name defined in the descriptor)")
 	flags.BoolVarP(&options.wait, "wait", "w", false, "Wait for cluster creation")
 	flags.IntVarP(&options.interval, "interval", "i", 10, "Interval in seconds for polling cluster status")
+	flags.StringVarP(&options.template, "template", "t", "", "Cluster template for creation")
```

Comment:
**Optional**: I think a hint about using template paths (relative or absolute) or alternatively keys (`eks`, `pke-on-aws`) for default templates **MAY** be very helpful here for first time users.